src/d3a/d3a_core/d3a_jobs.py

```diff
@@ -104,6 +104,9 @@ def start(scenario, settings, events, aggregator_device_mapping, saved_state):
             "aggregator_device_mapping": aggregator_device_mapping
         }
 
+        if d3a.constants.IS_CANARY_NETWORK:
+            config_settings['start_date'] = date.today()
```

Comment:
are you sure that it is using "UTC" timezone? Otherwise i would use pendulum.today and do
```suggestion
            config_settings['start_date'] = today(tz=TIMEZONE)
```